src/matches/matches.service.ts

```diff
@@ -54,15 +58,18 @@ export class MatchesService {
     return this.matchRepository.findOne({ where: { id: matchId } });
   }
 
-  async updateMatch(params: LeagueMatchParams, dto: UpdateMatchDto, leagueIdx: number, homeTeamIdx: number): Promise<Match> {
+  async updateMatch(params: LeagueMatchParams, dto: UpdateMatchDto, leagueIdx: number, homeTeamIdx: number, phoneNumber: string): Promise<Match> {
+    await this.cancelSMS(dto.observerSmsId)
```

Comment:
We need to handle this somehow, that is check if this action was successful and return something in case it wasn't. Generally 5** response codes are used for this. In case this request is not successful we could for example just pass on its response code received from callling the external api.